template/src/App.vue

```diff
@@ -1,35 +1,48 @@
 <template>
   <div id="app">
     <img class="logo" src="./assets/logo.png">
+    {{#if router}}
+    <router-view></router-view>
```

Comment:
To be honest, this is a good enough start with the vie-router integration. Sure, Hello is a placeholder component, but the `App` component is setup and people can replace the Hello component.

But I also agree with the complete setup path. Adding a template with vue-router, vuex, vue-mdl.